Covid19Radar/Covid19Radar/MultilingualResources/Covid19Radar.pt.xlf

```diff
@@ -693,7 +693,7 @@
         </trans-unit>
         <trans-unit id="SubmitConsentPageDescription5" translate="yes" xml:space="preserve">
           <source>If you have been in contact with you within 14 days, you will know that you may have been contacted without any information that can identify you personally.</source>
-          <target state="needs-review-translation" state-qualifier="mt-suggestion">Se você esteve em contato com você dentro de 14 dias, você saberá que você pode ter sido contatado sem qualquer informação que possa identificá-lo pessoalmente.</target>
+          <target state="needs-review-translation" state-qualifier="mt-suggestion">Se você esteve em contacto com você dentro de 14 dias, você saberá que você pode ter sido contatado sem qualquer informação que possa identificá-lo pessoalmente.</target>
```

Comment:
Replace `<target state="needs-review-translation" state-qualifier="mt-suggestion">` -> `<target state="translated">`

Reply:
That is not translated, how can you be in contact with yourself?

Reply:
GitHub is detecting changes. If you didn't intend to change it, don't change it.

Reply:
Search and replace thing, I'll revert that